_maps/icebox.json

```diff
@@ -41,8 +41,8 @@
          "Gravity":true,
          "Ice Ruins":true,
          "Weather_Snowstorm":true,
-         "Baseturf":"/turf/open/floor/plating/asteroid/snow/icemoon"
+         "Baseturf":"/turf/open/transparent/openspace/icemoon"
```

Comment:
Check it please
If you make hole over mining and it create `/turf/open/floor/plating/asteroid/snow/icemoon` and you get cold icemoon air, better place here `/turf/open/transparent/openspace`.

Reply:
I don't feel like people should be allowed to bomb their way into the top of ruins, if there was one with a complex puzzle that could be bypassed by just maxcapping the top of it that'd be a balance concern, so I'd prefer it keep the icemoon subtype.

Up to ya'll though.